modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,13 +4,25 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-APIs that have been protected by a Client ID enforcement policy require API consumers to create an application to hold the Client ID and Secret. This lets the app access the API as well as provide consumer analytics for an API developer.
+Requesting access registers a client and creates an API contract between an API
+and a client application.
+
+APIs that have been protected by a client ID enforcement policy require
```

Comment:
```suggestion
APIs that are protected by a client ID enforcement policy require
```

Reply:
I assume the repetition of this text is intentional since it's two different files. Good opportunity for an include.  :-)

Reply:
Good point. It is intentional, and it is a good opportunity for an include...